untrusted/test_enclave/src/lib.rs

```diff
@@ -0,0 +1,10 @@
+// Copyright (c) 2022 The MobileCoin Foundation
+#![allow(non_upper_case_globals)]
+#![allow(non_camel_case_types)]
+#![allow(non_snake_case)]
```

Comment:
Optional/Equivalent: Merge these
```suggestion
#![allow(non_camel_case_types, non_snake_case, non_upper_case_globals)]
```